lib/jsonapi_parameters/default_handlers/base_handler.rb

```diff
@@ -36,6 +36,8 @@ def included_object_base(included_object)
           end
 
           def client_generated_id?(related_id)
+            return false unless JsonApi::Parameters.client_id_prefix.present?
```

Comment:
simpy ruby version is better in this case (I don't like overusing activeSupport if nil is default)
```
            return false unless JsonApi::Parameters.client_id_prefix
```

Reply:
oh it's fixed already 👍